validate_email/smtp_check.py

```diff
@@ -224,41 +177,27 @@ def check(self, hosts: List[str]) -> bool:
             raise SMTPTemporaryError(error_messages=self.__temporary_errors)
 
 
-def mx_check(
-        email_address: EmailAddress, debug: bool,
+def smtp_check(
+        email_address: EmailAddress, mx_records: list, debug: bool,
         from_address: Optional[EmailAddress] = None,
-        helo_host: Optional[str] = None, smtp_timeout: int = 10,
-        dns_timeout: int = 10, skip_smtp: bool = False) -> bool:
+        helo_host: Optional[str] = None, smtp_timeout: int = 10) -> bool:
     """
-    Returns `True` as soon as the any server accepts the recipient
-    address.
+    Returns `True` as soon as the any of the given server accepts the
+    recipient address.
 
     Raise an `AddressNotDeliverableError` if any server unambiguously
     and permanently refuses to accept the recipient address.
 
     Raise `SMTPTemporaryError` if the server answers with a temporary
-    error code when validity of the email address can not be
-    determined. Greylisting or server delivery issues can be a cause for
-    this.
+    error code when validity of the email address can not be determined.
```

Comment:
Why the wrapping change here? As per PEP8, comment lines should be <72 columns long, this is why it was linebroken at before 'determined'.

Reply:
Sorry, I misunderstood the rule, I thought it was <= 72 columns. I can change this back, but then the next line would have to be broken before the "for", right?

Reply:
Quoting PEP8:
> The Python standard library is conservative and requires limiting lines to 79 characters (and docstrings/comments to 72).

Here's how it looks in my editor (I have a ruler set to 72 and 79 colums (I use Sublime Text):
![2021-03-07 12_52_15-~_Work_opensource_py3-validate-email_validate_email_smtp_check py (py3-validate-](https://user-images.githubusercontent.com/987055/110238814-9ab0ad00-7f3b-11eb-91e7-6f1d1ff35b63.png)

Reply:
Ok it's my bad, it seems it actually looks good, I got a bit confused here by looking at the webpage based diff. This is why I like to look at stuff in my editor, as you can see. :)